torch/_tensor_docs.py

```diff
@@ -2651,11 +2651,26 @@ def callable(a, b) -> number
 
 add_docstr_all('numpy',
                r"""
-numpy() -> numpy.ndarray
+numpy(*, force=False) -> numpy.ndarray
 
-Returns :attr:`self` tensor as a NumPy :class:`ndarray`. This tensor and the
+Returns :attr:`self` tensor as a NumPy :class:`ndarray`.
+
+If :attr:`force` is ``False``, tensor :attr:`self` and the
 returned :class:`ndarray` share the same underlying storage. Changes to
 :attr:`self` tensor will be reflected in the :class:`ndarray` and vice versa.
+If tensors storage is not CPU, or if :attr:`requires_grad` is ``True`` and
+:attr:`GradMode` is ``True``, or tensor is sparse, or :attr:`is_conj()`
+is ``True``, it throws exception.
+
+If :attr:`force` is ``True`` it tries to resolve issues, that would throw an exception.
```

Comment:
nit: extra "," in the middle of the sentence. Also "would otherwise throw" sounds a little better.

Reply:
What I meant is remove all the commas here :p